### ISSUE TITLE
Let JetFormBuilder Woocommerce Cart & Checkout plugin actually add to the cart and not delete everything

### ISSUE THREAD

**Issue by ninjamasterchris** [CrocoBlock/suggestions#6466]
The JetFormBuilder Woocommerce Cart & Checkout plugin now deletes all products from the cart and redirects to the checkout. That makes no sense at all. Please change it, so you can choose either: 

1. add to the cart 
2. replace the cart 

and then 

1. redirect to cart
2. redirect to checkout
3. redirect to any other page
4. show popup

That would make it a lot more useful I think.

Thanks!

**Comment by cwinkler8** [comment]
Yes! Please implement this as soon as possible. I have a JFB form that works great for calculating a dynamic price for an event and adding the required detail to my Woo Cart, but only for a single submission at a time. I’m currently forcing my users to checkout multiple times, once for each person attending an event. This is a terrible user experience, but also creates a lot of unnecessary transactions in PayPal.

**Comment by morosmo** [comment]
+1

**Comment by cwinkler8** [comment]
The current state of the add-on is not an "Add to Cart" it is a "Create new order" action. If there are any existing items in the cart, regardless of whether they were added via form or directly from a product, all items are cleared.

**Comment by cwinkler8** [comment]
@girafffee it looks like you've been added to a few tickets related to the Woo Cart add-on. Any chance there will be a release in the near future? This issue is really holding me up and I'd like to avoid paying for custom dev work if I can. Thanks!

**Comment by cwinkler8** [comment]
@girafffee are there any plans to work on the "Woo Add to Cart" add-on? I was able to get it add individual items to the cart and avoid the checkout redirect using a custom hook, however the add-on currently uses the last item added to describe the entire cart/order. It would be great to see this updated to allow an array for each individual item.
